docs/copied-from-beats/docs/template-config.asciidoc

```diff
@@ -23,9 +23,14 @@ endif::[]
 You can adjust the following settings to load your own template or overwrite an
 existing one.
 
-*`setup.template.enabled`*:: Set to false to disable template loading. If set this to false,
+*`setup.template.enabled`*:: Set to false to disable template loading. If this is set to false,
 you must <<load-template-manually,load the template manually>>.
 
+*`setup.template.type`*:: The type of template to use. Available options: `legacy` (default), index templates
+before Elasticsearch v7.8. Use this to avoid breaking existing deployments. New options are `component`
```

Comment:
@axw is this true also for us?

Reply:
Good catch. No, we should not document this for apm-server.

Reply:
Thank you! Will get this fixed.

Reply:
Fixed in c6c931ba4b0dcc05b0ddb4589467df54b76197b1.